Divyanshii-Divyanshi/Day 9- Calculate the nth term/Calculate the nth term.c

```diff
@@ -0,0 +1,28 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+
+int find_nth_term(int n, int a, int b, int c) 
+{
+  int i,arr[100];
+  arr[1]=a;
+  arr[2]=b;
+  arr[3]=c;
+  for(i=4;i<=n;i++)
```

Comment:
You were supposed to solve this problem using recursion, not by iterating through a loop.

Reply:
Okay sir, I'll keep this in mind and make the changes.